<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>120</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>색 그룹 추가</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="colorComboBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="colorNameText"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_colorNameAdd">
       <property name="text">
        <string>추가</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_colorNameRemove">
       <property name="text">
        <string>제거</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>401</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>색 아이템 추가</string>
   </property>
   <widget class="QListWidget" name="itemList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777213</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>27</y>
      <width>201</width>
      <height>52</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_colorItemSave">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>57</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_itemAdd">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>27</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_itemRemove">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>27</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
